CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ if (NOT CMAKE_BUILD_TYPE)
 endif()
 
 option(STATIC "Build statically-linked iotools?" ON)
+option(INSTALL_LINKS "Generate symlinks during install?" ON)
```

Comment:
Does it mean the default is on? I would move it `OFF` in this case (yes I know that initially it was like this, but again, I don't think distros will like this to be the default and I believe many of them will want to simply turn it `OFF`).